src/nni_manager/training_service/kubeflow/azureStorageClientUtils.ts

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright (c) Microsoft Corporation
+ * All rights reserved.
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+ * BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+'use strict';
+
+import * as fs from 'fs'
+import * as path from 'path';
+import { Deferred } from 'ts-deferred';
+import { getLogger } from '../../common/log';
+import { mkDirP } from '../../common/utils';
+
+export namespace AzureStorageClientUtility {
+    
+    /**
+     * create azure share
+     * @param fileServerClient 
+     * @param azureShare 
+     */
+    export async function createShare(fileServerClient: any, azureShare: any){
+        const deferred: Deferred<void> = new Deferred<void>();
+        fileServerClient.createShareIfNotExists(azureShare, function(error: any, result: any, response: any) {
+            if(error){
+                getLogger().error(`Create share failed:, ${error}`);
+                deferred.reject(error)
+            }else{
+                deferred.resolve()
+            }
+        })
+        return deferred.promise;
+    }
+    
+    /**
+     * create a directory in azure storage
```

Comment:
How about change the method annotation to:
Create a new directory (NOT recursively) in azure file storage
to keep consistent with createDirectory and can also be differentiated.